src/composable/ConditionalOrder.ts

```diff
@@ -19,10 +28,11 @@ import { ConditionalOrderArguments, ConditionalOrderParams, ContextFactory } fro
  * **NOTE**: Instances of conditional orders have an `id` property that is a `keccak256` hash of
  *           the serialized conditional order.
  */
-export abstract class ConditionalOrder<Data, Params> {
+export abstract class ConditionalOrder<D, S> {
```

Comment:
Now the params are `D` for user friendly data
`S` is the info from the struct that is encoded in `staticInput`